File: Ruby/proxybot/ProxyBot.rb
require 'socket'
include Socket::Constants
socket = Socket.new( AF_INET, SOCK_STREAM, 0 )
sockaddr = Socket.pack_sockaddr_in( 13337, '' )
socket.bind( sockaddr )
puts "Waiting for client..."
socket.listen( 5 )
client, client_sockaddr = socket.accept
puts "The client said, '#{client.readline.chomp}'"
#client.puts "Hello from script one!"
#socket.close

File: Python/proxybot/ProxyBot.py
# Echo server program
import socket

HOST = ''                 # Symbolic name meaning all available interfaces
PORT = 13337              # Arbitrary non-privileged port
s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
s.bind((HOST, PORT))
print 'Waiting for client...'
s.listen(1)
conn, addr = s.accept()
print 'Connected by', addr
data = conn.recv(1024)
if data: print 'Client sent: ', data
conn.close()
